test/choosetests.jl

```diff
@@ -151,7 +151,7 @@ function choosetests(choices = [])
 
     filtertests!(tests, "unicode", ["unicode/utf8"])
     filtertests!(tests, "strings", ["strings/basic", "strings/search", "strings/util",
-                   "strings/io", "strings/types"])
+                   "strings/io", "strings/types", "strings/tagged"])
```

Comment:
also test that `make -C test StyledStrings` works. I think there is a list of stdlibs in here somewhere.

Reply:
Yep, it does work. I see that a few of the tests to need to be updated though: I may have forgotten about them in the rename :sweat_smile:.

Reply:
```

00:28:02 ❯ make -C test StyledStrings
make: Entering directory '/home/tec/Documents/Projects/julia/test'
    JULIA test/StyledStrings
Running parallel tests with:
  nworkers() = 1
  nthreads() = 18
  Sys.CPU_THREADS = 20
  Sys.total_memory() = 31.147 GiB
  Sys.free_memory() = 14.820 GiB

Test      (Worker) | Time (s) | GC (s) | GC % | Alloc (MB) | RSS (MB)
StyledStrings  (1) |        started at 2023-10-06T00:28:05.044
StyledStrings  (1) |     1.17 |   0.01 |  0.9 |     115.34 |   385.13

Test Summary: | Pass  Total  Time
  Overall     |   84     84  2.1s
    SUCCESS
make: Leaving directory '/home/tec/Documents/Projects/julia/test'
```

There we go :slightly_smiling_face: